src/test/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/resource/KeywordSearchResourceTest.kt

```diff
@@ -195,6 +196,52 @@ class KeywordSearchResourceTest : QueueIntegrationTest() {
     )
   }
 
+  @Test
+  fun `default order is by score, prison number`() {
+    keywordSearch(
+      keywordRequest = KeywordRequest(
+        andWords = "jones",
+        prisonIds = listOf("MDI", "AGI", "LEI"),
+        type = SearchType.DEFAULT
```

Comment:
Maybe leave this off the request to demonstrate that not passing a type defaults to SearchType.DEFAULT?